L100-core-narrow-call-details.md

```diff
@@ -0,0 +1,36 @@
+L100: C-Core Narrow grpc_call_details
+----
+* Author(s): ctiller
+* Approver: markdroth
+* Status: In Review
+* Implemented in: C Core
+* Last updated: [Date]
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Please create the discussion thread and add a link here.  That starts to 10-business-day clock for merging this gRFC.